lib/core/public/cleanup-plugins.js

```diff
@@ -23,14 +23,18 @@ function cleanupPlugins(resolve, reject) {
     });
   });
 
-  axe.utils.toArray(document.querySelectorAll('frame, iframe')).forEach(function (frame) {
+  var flattenedTree = axe.utils.getFlattenedTree(document.body)

```

Comment:
I had to create a new flattened tree to support this functionality. The only other place in which a flattened tree is created is during the audit itself. Since `axe.cleanup` can be called at any arbitrary time, we can't reuse the same tree.